invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -0,0 +1,185 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import { Modal, Message, Grid, Form, Checkbox, Button } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { Formik } from "formik";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { SuccessIcon } from "@js/invenio_communities/members";
+import { RadioField, TextAreaField, http, withCancel } from "react-invenio-forms";
+import * as Yup from "yup";
+import _get from "lodash/get";
+
+export class AccessRequestsTab extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: undefined,
+    };
+  }
+
+  componentWillUnmount() {
+    this.cancellableAction && this.cancellableAction.cancel();
+  }
+
+  accessRequestSchema = Yup.object({
+    allow_user_requests: Yup.bool(),
+    allow_guest_requests: Yup.bool(),
+    accept_conditions_text: Yup.string().nullable(),
+  });
+
+  handleSubmit = async (values) => {
+    this.setState({ loading: true, actionSuccess: false });
+    const { record, successCallback } = this.props;
+    const payload = {
+      access: {
+        settings: {
+          allow_user_requests: values.allow_user_requests,
+          allow_guest_requests: values.allow_guest_requests,
+          accept_conditions_text: values.accept_conditions_text,
+        },
+      },
+    };
+    this.cancellableAction = withCancel(
+      http.put(record.links.self + "/access", payload)
```

Comment:
this should be generated from the backend no?

Reply:
There is only the `/records/ehhv6-7jt48/access/links` in `access_links` link